content/en/docs/Examples/services.md

```diff
@@ -0,0 +1,409 @@
+    web_url: https://goauthentik.io/docs/releases/{{ version | split:"." | slice:":-1" | join:"."  }}
+    icon: https://raw.githubusercontent.com/goauthentik/authentik/master/web/icons/icon.png
+    deployed_version:
+      url: https://authentik.example.io/api/v3/admin/version/
+      headers:
+        - key: Authorization
+          value: bearer <API Token>
+      json: version_current
+```
+
+## golang/go
+Source: https://github.com/golang/go
+```yaml
+services:
+  golang/go:
+    type: url
+    url: https://golang.org/dl/
+    url_commands:
+      - type: regex_submatch
+        regex: go([0-9.]+[0-9]+)\.src\.tar\.gz
+    web_url: https://go.dev/doc/go{{ version | split:"." | slice:":2" | join:"."  }}
+    semantic_versioning: false
+    icon: https://go.dev/images/gophers/motorcycle.svg
+```
+
+## gohugoio/hugo
+Source: https://github.com/gohugoio/hugo
+```yaml
+services:
+gohugoio/hugo:
+    type: github
+    url: gohugoio/hugo
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://github.com/gohugoio/hugo/releases
+    regex_content: hugo_{{ version }}_Linux-64bit\.deb
+    icon: https://raw.githubusercontent.com/gohugoio/hugo/master/docs/static/img/hugo.png
+```
+
+## gotify/server
+Source: https://github.com/gotify/server
+```yaml
+services:
+  gotify/server:
+    type: github
+    url: gotify/server
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://github.com/gotify/server/releases
+    icon: https://github.com/gotify/logo/raw/master/gotify-logo.png
+```
+
+## grafana/grafana
+Source: https://github.com/grafana/grafana
+```yaml
+service:
+  grafana/grafana:
+    type: github
+    url: grafana/grafana
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://grafana.com/docs/grafana/latest/release-notes/release-notes-{{ version | split:"." | join:"-"  }}
+    icon: https://cdn.worldvectorlogo.com/logos/grafana.svg
+    deployed_version:
+      url: https://grafana.example.io/login
+      regex: v([0-9.]+)\s\([0-9a-z]+\)
+```
+
+## grafana/loki
+Source: https://github.com/grafana/loki
+```yaml
+services:
+  grafana/loki:
+    type: github
+    url: grafana/loki
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://grafana.com/docs/loki/latest/release-notes/v{{ version | split:"." | slice:":2" | join:"-"  }}
+    icon: https://grafana.com/static/assets/img/blog/loki.png
+```
+
+## healthchecks/healthchecks
+Source: https://github.com/healthchecks/healthchecks
+```yaml
+services:
+  healthchecks/healthchecks:
+    type: github
+    url: healthchecks/healthchecks
+    url_commands:
+      - type: regex_submatch
+        regex: v([0-9.]+)$
+    web_url: https://github.com/mailcow/mailcow-dockerized/releases/tag/{{ version }}
+    icon: https://raw.githubusercontent.com/mailcow/mailcow-dockerized/master/data/web/img/cow_mailcow.svg
```

Comment:
Unrelated icon and web_url

Reply:
web_url is changed to the right project.

The icon is a bit more difficult as there is no icon in the repo itself, only a placeholder.